core-web/libs/block-editor/src/lib/extensions/image-tabview-form/image-tabview-form.component.html

```diff
@@ -0,0 +1,32 @@
+<div class="tabview-container">
+    <ng-container [formGroup]="form">
+        <p-tabView>
+            <p-tabPanel header="dotCMS" leftIcon="pi pi-images">
+                <div class="search-box">
+                    <span class="p-input-icon-right">
+                        <input
+                            class="search"
+                            #inputSearch
+                            type="text"
+                            pInputText
+                            formControlName="search"
+                            placeholder="Search"
+                        />
+                        <i class="pi pi-search"></i>
+                    </span>
+                </div>
+                <ng-container *ngIf="contentlets$ | async as contentlets">
+                    <dot-image-card-list
+                        [contentlets]="contentlets"
+                        [done]="preventScroll"
+                        [loading]="loading"
+                        (selectedItem)="selectItemCallback($event)"
+                        (nextBatch)="offset$.next($event)"
+                    ></dot-image-card-list>
+                </ng-container>
+            </p-tabPanel>
+            <!-- <p-tabPanel header="Upload" leftIcon="pi pi-upload"> Content 2 </p-tabPanel> -->
```

Comment:
Remove comments

Reply:
@oidacra These tabs are commented because they are part of other issues (https://github.com/dotcms/core/issues/23237, https://github.com/dotcms/core/issues/23238)